api/apiFunctions.ts

```diff
@@ -0,0 +1,17 @@
+import { account } from "./config"
+
+export async function loginAccount(user: { email: string; password: string }) {
```

Comment:
Just curious, is there a reason you added a comment for logoutAccount and not loginAccount?